web/src/locales/sv.json

```diff
@@ -134,7 +132,6 @@
     },
     "estimated_time_slicer_average": {
       "title": "Data är för närvarande uppskattad",
-      "pill": "Otillgänglig",
```

Comment:
I wonder if we should just add this key to the en file instead?

Reply:
I think it's okay to remove for now. We should re-add this with translations from "Preliminary" in the future, English also needs this for the badges in AVO-282. I'll add some context there